dap_prinz_green_jobs/analysis/ojo_analysis/aggregate_by_soc_part_2.py

```diff
@@ -0,0 +1,51 @@
+"""
+Small script to merge the outputs of aggregate_by_soc.py with occupation_similarity.py
+"""
```

Comment:
I would just add this all to aggregate_by_soc.py instead of having it as a separate script?

Reply:
Good shout!